js/reverse.js

```diff
@@ -0,0 +1,14 @@
+const reverse = (text) => {
```

Comment:
- **_[Optional] Consider using the name of the function(`reverseString`) specified in the project specification_**